packages/@react-spectrum/actiongroup/test/ActionGroup.test.js

```diff
@@ -279,7 +278,7 @@ describe('ActionGroup', function () {
     let {getByTestId} = render(
       <Provider theme={theme} locale="de-DE">
         <ActionGroup>
-          <ActionButton UNSAFE_className={'test-class'} icon={<Brush />} data-testid="button-1">Click me</ActionButton>
+          <ActionButton UNSAFE_className={'test-class'} data-testid="button-1">Click me</ActionButton>
```

Comment:
did you mean to add the icon as a child?

Reply:
the icon didn't appear to be used in the test...